lib/src/exporter.dart

```diff
@@ -7,6 +7,8 @@ import 'package:screen_recorder/src/frame.dart';
 
 class Exporter {
   final List<Frame> _frames = [];
+  List<Frame> get frames => _frames;
```

Comment:
Is the intention with this to modify the frames before export, so that the edited frames are exported? Or is the intent to do something other with it?
Then you might want to wrap the list in an unmodifiable list.

Reply:
@ueman  I am creating a CustomExporter by extending the base Exporter, but i can not totally do it without the frames public. I did not want to introduce more complexity to the existing exporter since on my custom one I'm using many things not related to the package like some state management solution.

Reply:
Gotcha. Is there a possibility to see what you're doing with this package? I would love to see what you've done with it.

Reply:
So far the project is private so unfortunately i can't show anything :(